plottable.js

```diff
@@ -12698,6 +12347,90 @@ var Plottable;
+/// <reference path="scales/colorScale.ts" />
+/// <reference path="scales/timeScale.ts" />
+/// <reference path="scales/interpolatedColorScale.ts" />
+/// <reference path="scales/tickGenerators.ts" />
+/// <reference path="drawers/drawer.ts" />
+/// <reference path="drawers/lineDrawer.ts" />
+/// <reference path="drawers/areaDrawer.ts" />
+/// <reference path="drawers/rectangleDrawer.ts" />
+/// <reference path="drawers/arcDrawer.ts" />
+/// <reference path="drawers/arcOutlineDrawer.ts" />
+/// <reference path="drawers/symbolDrawer.ts" />
+/// <reference path="drawers/segmentDrawer.ts" />
+/// <reference path="components/component.ts" />
+/// <reference path="components/componentContainer.ts" />
+/// <reference path="components/group.ts" />
+/// <reference path="axes/axis.ts" />
+/// <reference path="axes/timeAxis.ts" />
+/// <reference path="axes/numericAxis.ts" />
+/// <reference path="axes/categoryAxis.ts" />
+/// <reference path="components/label.ts" />
+/// <reference path="components/legend.ts" />
+/// <reference path="components/interpolatedColorLegend.ts" />
+/// <reference path="components/gridlines.ts" />
+/// <reference path="components/table.ts" />
+/// <reference path="components/selectionBoxLayer.ts" />
+/// <reference path="components/guideLineLayer.ts" />
+/// <reference path="plots/plot.ts" />
+/// <reference path="plots/piePlot.ts" />
+/// <reference path="plots/xyPlot.ts" />
+/// <reference path="plots/rectanglePlot.ts" />
+/// <reference path="plots/scatterPlot.ts" />
+/// <reference path="plots/barPlot.ts" />
+/// <reference path="plots/linePlot.ts" />
+/// <reference path="plots/areaPlot.ts" />
+/// <reference path="plots/clusteredBarPlot.ts" />
+/// <reference path="plots/stackedAreaPlot.ts" />
+/// <reference path="plots/stackedBarPlot.ts" />
+/// <reference path="plots/segmentPlot.ts" />
+/// <reference path="plots/waterfallPlot.ts" />
+/// <reference path="plots/wheelPlot.ts" />
+/// <reference path="animators/animator.ts" />
+/// <reference path="animators/nullAnimator.ts" />
+/// <reference path="animators/easingAnimator.ts" />
+/// <reference path="dispatchers/dispatcher.ts" />
+/// <reference path="dispatchers/mouseDispatcher.ts" />
+/// <reference path="dispatchers/touchDispatcher.ts" />
+/// <reference path="dispatchers/keyDispatcher.ts" />
+/// <reference path="interactions/interaction.ts" />
+/// <reference path="interactions/clickInteraction.ts" />
+/// <reference path="interactions/doubleClickInteraction.ts" />
+/// <reference path="interactions/keyInteraction.ts" />
+/// <reference path="interactions/pointerInteraction.ts" />
+/// <reference path="interactions/panZoomInteraction.ts" />
+/// <reference path="interactions/dragInteraction.ts" />
+/// <reference path="components/dragBoxLayer.ts" />
+/// <reference path="components/xDragBoxLayer.ts" />
+/// <reference path="components/yDragBoxLayer.ts" />
+/// <reference path="components/dragLineLayer.ts" />
+///<reference path="../reference.ts" />
```

Comment:
I'm unsure why this got printed here.  Was trying to investigate it to no avail.  Note that this ultimately doesn't affect anything, but it is sad to see in the build.